### ISSUE TITLE
idea: alternative user interface

### ISSUE THREAD

**Issue by severak** [klknn/kdr#1]
there was some discussion ([at KVR](https://www.kvraudio.com/forum/viewtopic.php?t=543132)) about Synth1 that it has somewhat confusing user interface.

One guy even designed [alternative interface](https://www.samdetomasi.com/synth1-ui-redesign/) for it. It may be a good idea to copy some of his ideas (namely aligning controls to the grid).

**Comment by klknn** [comment]
Yes, the synth1 UI is confusing for me too. Currently, UI is the last item in my TODO list but such good ideas are welcome. Thanks.
